setup.py

```diff
@@ -29,15 +29,14 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        'click==7.0',
+        'click>=7.1.2',
```

Comment:
Why not support `7.0`, do we use something that is only available in 7.1.2 and higher?

Reply:
Changed to `click`, which means that `>=8.0` will be used.